index.js

```diff
@@ -23,13 +25,12 @@ program
 
 if (program.license) {
   const cwd = process.cwd();
-  const licenseFile = __dirname + '/licenses/' + program.license + '.txt';
-  fs.createReadStream(licenseFile).pipe(fs.createWriteStream(cwd + '/LICENSE')); 
```

Comment:
Also took out this copy stream, I think the readFile followed by the writeFile should be sufficient here